update_copyright_hook

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+in1="^.*?This is software[.\s\S]+?\)\."
+out1="This is software developed originally for the Collaborative Computational
+Project in Positron Emission Tomography and Magnetic Resonance Imaging
+(http://www.ccppetmr.ac.uk/) and now as part of CCP-SyneRBI."
+
+# Stash unstaged changes
+git stash -q --keep-index
+
+readonly FILES=$(git diff --diff-filter=ACMRTUXB --cached --name-only)
+
+for f in $FILES; do
+	perl -0777 -i -pe "s#${in1}#${out1}#g" $f
```

Comment:
if possible, don't rely on `perl`. `sed` could do the same thing?

Reply:
I read that `sed` isn't so good for multiple line replacements. I have no preference for `perl` if it's possible to use `sed`.

Reply:
as an aside `sed` works slightly differently on a Mac.

However it would be better to write the whole thing in python rather than `bash` to include @evgueni-ovtchinnikov & other win people.